archlinux/Makefile

```diff
@@ -0,0 +1,102 @@
+#rootfs for Archlinux ARM
+
+SRCDIR=src
+BUILDDIR=build
+CUSTOMIZATION=customization
+
+SUDO=/usr/bin/sudo
+QEMU=/usr/bin/qemu-arm-static
+QEMU64=/usr/bin/qemu-aarch64-static
+
+GNULINUX_OTA_ARCH=armv7
```

Comment:
Yes sure for all with GNULINUX_ but for SUDO , QEMU, SRCDIR etc, those are really not linked with archlinux so I prefer to keep them without if it is fine ?